packages/mobile/src/account/DollarEducation.test.tsx

```diff
@@ -28,29 +14,4 @@ describe('DollarEducation', () => {
     )
     expect(tree).toMatchSnapshot()
   })
-
-  describe('#goToSend', () => {
```

Comment:
These tests weren't working anymore but what they test is so trivial it's not worth the time to fix them. Ditto for the other education.test.tsx files.